vulcanai2/plotters/visualization.py

```diff
@@ -49,17 +38,24 @@ def save_visualization(plot, path=None):
     None
 
     """
-    if not assert_display_available() and path is None:
-        raise RuntimeError("No display environment found. Display environment needed to plot, or set save_path=path/to/dir")
-    else:
-        plot.savefig(path)
+    plot.savefig(path)
+
+def get_save_path(path, vis_type):
+    path = path + "{}_{date:%Y-%m-%d_%H:%M:%S}".format(vis_type, date=datetime.now()) + '.png'
+    return path
 
 def get_time():
     curr_time = str(datetime.now())
     curr_time = curr_time.replace(" ", '_')
     curr_time = curr_time.split('.')[0]
     return curr_time
 
+def load_vis(load_path=None):
```

Comment:
@nirtiac @joemehltretter should we pass it a network file path to load and then use basenetwork.load_model to get the record/display it?

Reply:
@rfratila so something along the lines of `model = basenetwork.load_model(model_path)` then send `model.record` to `display_record`?

Reply:
Yeah something like that

Reply:
@rfratila @nirtiac We can't use `basenetwork.load_model` due to circular dependency issues. I going to remove the load function and push that. Then I'll create an issue regarding this for further discussion.